release/models/types/openconfig-inet-types.yang

```diff
@@ -154,10 +160,10 @@ module openconfig-inet-types {
     type string {
       pattern
         '([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\.([0-9]|'
-        + '[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])){3}(%[a-zA-Z0-9_]+)';
+        + '[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])){3}(%[a-zA-Z0-9_.-/:]+)';
       oc-ext:posix-pattern
         '^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\.([0-9]|'
-        + '[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])){3}(%[a-zA-Z0-9_]+))$';
+        + '[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])){3}(%[a-zA-Z0-9_.-/:]+))$';
```

Comment:
This pattern looks like it'll allow `IP_ADDRESS%:.-` which doesn't seem like a valid interface name. 

Would a regex like `%[a-zA-Z0-9_]([.-/:][a-zA-Z0-9_]+)?` be more appropriate?